### ISSUE TITLE
is there any support for this code ?

### ISSUE THREAD

**Issue by nimaaghli** [v01d/ardrone-proxy#1]
Hello, 
i have some problems making this proxy im getting i have installed all boost dev tools  :
/usr/lib/gcc/arm-linux-gnueabi/4.6/../../../../arm-linux-gnueabi/bin/ld: cannot find -lboost_system
i want to use you code to pass commands from arduino to ardrone , have ever tested something like that ?

**Comment by v01d** [comment]
Hi, I'm not really maintaining this anymore. However, I just pushed a small change which may fix this. Please try it.

**Comment by nimaaghli** [comment]
Thank You for modification ,
I think your code is big help for those who want to do on-board autonomous flight and im surprised i was the first one asking question about your code .


I would very grateful if you can give some information about building code after spending about 10 hour im not able to compile this there are some errors due to changes in mavlink i changed them in source code but which version of your Boost you used with code ? each version im installing give me different error from headers because they are changed or removed in different version.

**Comment by nimaaghli** [comment]
ok now im here :
[ 20%] Built target mavlink_headers
[ 40%] Building CXX object CMakeFiles/ardrone-proxy.dir/serial.cpp.o
[ 60%] Building CXX object CMakeFiles/ardrone-proxy.dir/network.cpp.o
[ 80%] Building CXX object CMakeFiles/ardrone-proxy.dir/protocol.cpp.o
make[2]: *** No rule to make target `/usr/lib/libboost_system-mt.so', needed by `ardrone-proxy'.  Stop.
make[1]: *** [CMakeFiles/ardrone-proxy.dir/all] Error 2
make: *** [all] Error 2

I will compile boost tomorrow to generate .so files the thing is that do i need to cross compile boost using this :
http://www.boost.org/build/doc/html/bbv2/tasks/crosscompile.html
or just compile it for x86 ?
thanks for your helps

**Comment by v01d** [comment]
Yes, you need to cross-compile.

**Comment by nimaaghli** [comment]
thank it done i put the file in ardrone it gave this :+1:
Starting ardrone-proxy
sending process to background
is it means the program is listening in background ? i couldnt see the name of it in list by top command to see list of processes

**Comment by v01d** [comment]
Yes, it should be running. Read the readme file please, this is documented.
